fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -55,10 +55,10 @@
       http(method, url, headers, data, function(res) {
         if (res.status == 201) {
           const loc = res.getResponseHeader('Location');
-          const link = res.getResponseHeader('Link');
-          if (link != null && link.match('rel="describedby"')) {
-            const str = link.match(/<[^>]+>/)[0];
-            window.location = str.slice(1, str.length - 1);
+          const linkheaders = res.getResponseHeader('Link').split(", ");
```

Comment:
I am hitting a `null` in this line when creating a new resource container resource.